Gruntfile.js

```diff
@@ -406,338 +407,340 @@ module.exports = function(grunt) {
 //         separator: ';\n\n'
       },
       MMLSVG: {
+        nonull: true,
```

Comment:
This is an option on `grunt-contrib-concat` plugin.
It's a strange name, I would agree.
Without this option, any missing file will fail silently.
I have turned this on so I can see warnings when some of the files in the list are not present. This is how I found some missing files that I commented out in the lists.

Documentation here: https://github.com/gruntjs/grunt-contrib-concat
Lookup the `nonull` option in the README file.